source/helpers/Objects.tsx

```diff
@@ -44,10 +45,10 @@ const excludePropetiesWithKey = (object = {}, keys = []) =>
  * @param { object } object Target to change
  * @param { match } match String to match
  */
-const renameMatchedKeysInObject = (obj = {}, match = '') => {
-  Object.keys(obj).forEach(key => {
+const renameMatchedKeysInObject = (obj = {}, match = "") => {
```

Comment:
Not sure if there's a reason, most of this file is not part of this PR and was written beforehand by someone else. The reason they're probably marked as changed is because of automatic styling changes, but I can fix the inconsistencies while I'm at it 🙂.